templates/zerver/login.html

```diff
@@ -113,7 +113,8 @@ <h1 class="get-started">{{ _("Log in to Zulip") }}</h1>
                         <form class="social_login_form form-inline" action="{{ backend.login_url }}" method="get">
                             <input type="hidden" name="next" value="{{ next }}">
                             <button id="login_{{ backend.button_id_suffix }}" class="login-social-button"
-                              {% if backend.display_icon %} style="background-image:url({{ backend.display_icon }})"  {% endif %}>
+                              {% if backend.display_name == "Apple" %} style="background-image: url(https://appleid.cdn-apple.com/appleid/button/?width=328&height=46&color=white&locale={{locale}});"{% endif %}>
+                                {% if backend.display_icon %} style="background-image:url({{ backend.display_icon }})"  {% endif %}>
```

Comment:
Shouldnt this be `{% elif backend.display_icon %}`? Could you also show a screenshot of the button?

Reply:
ah, I messed it up because I was having linting issues as `<button` closing tag`>` wasn't set properly. Now I've edited it, but had to set `font-size: 0px;` in css for apple button.